modules/combined/tests/eigenstrain/composite.i

```diff
@@ -5,6 +5,26 @@
   ny = 5
 []
 
+[GlobalParams]
```

Comment:
Is it really necessary to add all of the mechanics stuff to this test?  Maybe I'm missing something, but it seems like just removing all the the `execute_on = initial` lines like you did should be enough to get this test to pass.